src/main/scala/zio/prelude/AssociativeBoth.scala

```diff
@@ -1034,18 +985,6 @@ object AssociativeBoth extends LawfulF.Invariant[AssociativeBothDeriveEqualInvar
         fa.flatMap(a => fb.map(b => (a, b)))
     }
 
-  /**
-   * The `AssociativeBoth` instance for `Option`.
-   */
-  implicit val OptionAssociativeBoth: AssociativeBoth[Option] =
```

Comment:
It's good that you removed this redundant instance `OptionAssociativeBoth`.
But now you need to move `OptionIdentityBoth` from `IdentityBoth` to here (`AssociativeBoth`), so that Scala's implicit resolution mechanism can find it. And analogously for the other instances.

Reply:
Thanks for explanation. Fixed.